gen-tests.gradle

```diff
@@ -149,23 +149,36 @@ task generateKotlinTests(type: JavaExec) {
   classpath = configurations.wire
   main = 'com.squareup.wire.WireCompiler'
   args = [
-      '--proto_path=wire-tests/src/commonTest/proto/kotlin',
-      '--kotlin_out=wire-tests/src/commonTest/proto-kotlin',
-      'all_types.proto',    
-      'person.proto',
-      'optional_enum.proto',
-      'map.proto',
-      'one_of.proto',
-      'same_name_enum.proto',
-      'service_kotlin.proto',
-      'service_without_package.proto',
-      'redacted_test.proto',
-      'redacted_one_of.proto',
-      'deprecated_enum.proto',
-      'deprecated.proto',
-      'form.proto',
-      'no_fields.proto',
-      'to_string.proto'
+          '--proto_path=wire-tests/src/commonTest/proto/kotlin',
+          '--kotlin_out=wire-tests/src/commonTest/proto-kotlin',
+          'all_types.proto',
+          'person.proto',
+          'optional_enum.proto',
+          'map.proto',
+          'one_of.proto',
+          'same_name_enum.proto',
+          'redacted_test.proto',
+          'redacted_one_of.proto',
+          'deprecated_enum.proto',
+          'deprecated.proto',
+          'form.proto',
+          'no_fields.proto',
+          'to_string.proto'
+  ]
+}
+
+// KOTLIN SERVICES
+
+task generateKotlinServicesTests(type: JavaExec) {
```

Comment:
It's probably worth creating a separate source set for these tests, can help with that if needed.

Reply:
maybe in follow up